js/libs/keycloak-admin-client/src/defs/userProfileMetadata.ts

```diff
@@ -1,10 +1,37 @@
-// See: https://github.com/keycloak/keycloak/blob/main/core/src/main/java/org/keycloak/representations/idm/UserProfileMetadata.java
-export interface UserProfileMetadata {
-  attributes?: UserProfileAttributeMetadata[];
-  groups?: UserProfileAttributeGroupMetadata[];
+export default interface UserProfileConfig {
```

Comment:
The `UserProfileConfig` is a different type of REST API response than `UserProfileMetadata`, and should be contained in in it's own file. I don't recall the exact details, but I believe that  `UserProfileMetadata` is supposed be used generically, whereas `UserProfileConfig` is exclusively used for administration purposes.

@pedroigor is the above correct?

Reply:
I don't see how / why the filename matters I could rename it to `userProfile.ts`, to me it makes sense to have it all in one file

Reply:
Mostly because they are two separate representations that are meant for different purposes, and should not be used interchangeably. Putting them in the same file can lead people touching this file in the future to believe they might be the same thing.